parsers/uniprot/xml.go

```diff
@@ -0,0 +1,585 @@
+// Code generated by xsdgen. DO NOT EDIT.
+
+package uniprot
+
+import (
+	"bytes"
+	"encoding/xml"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Absorption struct {
+	Max  EvidencedStringType   `xml:"http://uniprot.org/uniprot max,omitempty"`
+	Text []EvidencedStringType `xml:"http://uniprot.org/uniprot text,omitempty"`
+}
+
+type AlternativeName struct {
+	FullName  EvidencedStringType   `xml:"http://uniprot.org/uniprot fullName,omitempty"`
+	ShortName []EvidencedStringType `xml:"http://uniprot.org/uniprot shortName,omitempty"`
+	EcNumber  []EvidencedStringType `xml:"http://uniprot.org/uniprot ecNumber,omitempty"`
+}
+
+type Anon6 struct {
+	Type Type `xml:"type,attr"`
+}
+
+// Describes different types of citations.
+// Equivalent to the flat file RX-, RG-, RA-, RT- and RL-lines.
+type CitationType struct {
+	Title       string            `xml:"http://uniprot.org/uniprot title,omitempty"`
+	EditorList  NameListType      `xml:"http://uniprot.org/uniprot editorList,omitempty"`
+	AuthorList  NameListType      `xml:"http://uniprot.org/uniprot authorList,omitempty"`
+	Locator     string            `xml:"http://uniprot.org/uniprot locator,omitempty"`
+	DbReference []DbReferenceType `xml:"http://uniprot.org/uniprot dbReference,omitempty"`
+	Type        Type              `xml:"type,attr"`
+	Date        Date              `xml:"date,attr,omitempty"`
+	Name        string            `xml:"name,attr,omitempty"`
+	Volume      string            `xml:"volume,attr,omitempty"`
+	First       string            `xml:"first,attr,omitempty"`
+	Last        string            `xml:"last,attr,omitempty"`
+	Publisher   string            `xml:"publisher,attr,omitempty"`
+	City        string            `xml:"city,attr,omitempty"`
+	Db          string            `xml:"db,attr,omitempty"`
+	Number      string            `xml:"number,attr,omitempty"`
+	Institute   string            `xml:"institute,attr,omitempty"`
+	Country     string            `xml:"country,attr,omitempty"`
+}
+
+// Describes a cofactor.
```

Comment:
```suggestion
```
redundantt

Reply:
xml.go is automatically generated, so I don't think we should edit it.